bindings/rust/bench/benches/handshake.rs

```diff
@@ -29,10 +30,14 @@ pub fn bench_handshake_params(c: &mut Criterion) {
                         },
                         handshake_type,
                     )
-                    .unwrap()
                 },
                 |harness| {
-                    harness.handshake().unwrap();
+                    // if harness invalid, do nothing but don't panic
+                    // useful for historical performance bench to ignore configs
+                    // invalid only for past versions of s2n-tls
```

Comment:
```suggestion
                    // Ignore failure since some past versions of s2n-tls have a different API and therefore
                    // fail to build. Since the results are graphed as part of historical perf, its possible to
                    // extrapolate any missing data.
```

Reply:
This fail-safe part of the code isn't because of API changes causing build failure, but rather API changes that cause a runtime error (namely security policies that were only added recently).